tests/e2e/v2store_deprecation_test.go

```diff
@@ -125,15 +129,12 @@ func TestV2DeprecationSnapshotMatches(t *testing.T) {
 	assert.NoError(t, epc.Close())
 
 	assertSnapshotsMatch(t, oldMemberDataDir, newMemberDataDir, func(data []byte) []byte {
-		// Patch cluster version
```

Comment:
Why remove this?

Reply:
please refer to the comment above - https://github.com/etcd-io/etcd/pull/16418#issuecomment-1679344123.